Application/EdFi.Ods.Features/Composites/Infrastructure/HqlBuilder.cs

```diff
@@ -788,7 +788,7 @@ private void ProcessQueryStringParameters(HqlBuilderContext builderContext, Comp
                     {
                         var id = _descriptorsCache.GetId(
                             targetProperty.LookupTypeName,
-                            (string) queryStringParameter.Value);
+                            Convert.ToString(queryStringParameter.Value));
```

Comment:
Why the change from the cast?

Reply:
(string) queryStringParameter.Value' threw an exception of type 'System.InvalidCastException'
    Data: {System.Collections.ListDictionaryInternal}
    HResult: -2147467262
    HelpLink: null
    InnerException: null
    Message: "Specified cast is not valid."
    Source: "3758471d3c4b41068e3d4cce19464a90"
    StackTrace: "   at <>x.<>m0(HqlBuilder <>4__this, HqlBuilderContext builderContext, CompositeDefinitionProcessorContext processorContext)"
    TargetSite: {System.String <>m0(EdFi.Ods.Features.Composites.Infrastructure.HqlBuilder, EdFi.Ods.Features.Composites.Infrastructure.HqlBuilderContext, EdFi.Ods.Features.Composites.CompositeDefinitionProcessorContext)}

Reply:
So  I changed to  Convert.ToString(queryStringParameter.Value) to fix this issue